src/js/TuningSliders.js

```diff
@@ -387,14 +410,20 @@ TuningSliders.updateFormPids = function(updateSlidersOnly = false) {
         $('.pid_tuning .YAW input[name="f"]').val(FC.ADVANCED_TUNING.feedforwardYaw);
     }
 
-    $('output[name="sliderMasterMultiplier-number"]').val(this.sliderMasterMultiplier);
-    $('output[name="sliderRollPitchRatio-number"]').val(this.sliderRollPitchRatio);
-    $('output[name="sliderIGain-number"]').val(this.sliderIGain);
-    $('output[name="sliderPDRatio-number"]').val(this.sliderPDRatio);
-    $('output[name="sliderPDGain-number"]').val(this.sliderPDGain);
-    $('output[name="sliderDMinRatio-number"]').val(this.sliderDMinRatio);
-    $('output[name="sliderFeedforwardGain-number"]').val(this.sliderFeedforwardGain);
-
+    if (semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
+        $('output[name="sliderDGain-number"]').val(this.sliderDGain);
+        $('output[name="sliderPIGain-number"]').val(this.sliderPIGain);
+        $('output[name="sliderFeedforwardGain-number"]').val(this.sliderFeedforwardGain);
```

Comment:
Same here, is aplied to all versions and can go outside.

Reply:
Fixed.